source/support.tex

```diff
@@ -1210,32 +1210,53 @@
   using uint_least64_t = @\textit{unsigned integer type}@;
 
   using uintmax_t      = @\textit{unsigned integer type}@;
-  using uintptr_t      = @\textit{unsigned integer type}@; // optional
-} // namespace std
-\end{codeblock}
+  using uintptr_t      = @\textit{unsigned integer type}@;  // optional
+}
 
-\pnum
-The header also defines numerous macros of the form:
+#define INT[8, 16, 32, 64]_MIN @\seebelow@         // optional
```

Comment:
I will open an LWG issue to have the question settled of whether the macros are optional.